### PR TITLE
remove qm externals access

### PR DESCRIPTION
**Pull request by Emisse**
people are using this as argument to give cmo externals access

### CODE REVIEW

**Comment by nikthechampiongr** [comment]
Doesn't the QM have external access because they oversee salv? It seems pretty reasonable for them to have it when they are supposed to supervise people who do their jobs entirely in space. The cmo does not supervise people who spent most of their time in space.

**Comment by Emisse** [comment]
the cargo bay doors dont use externals access i dont see the need for them to have it

**Comment by nikthechampiongr** [comment]
Ok then.

**Comment by UbaserB** [comment]
dont cargo techs also have externals access?

**Comment by LordEclipse** [comment]
Actually, CMO is the only head without externals. It would be funny when a cargo tech can open externals but their own boss can't.

**Comment by Emisse** [comment]
RD does not have externals

**Comment by LordEclipse** [comment]
Wait really? That's news to me. So that's 2 heads, but RD and CMO don't work near space at all, whereas QM works directly with space. Again it would be pretty hilarious for a QM to get shoved out of a shuttle or the airlock and not being able to simply enter the station through the airlocks he works right next to, especially when Cargo Techs would be able to. So I'm 50/50 on this, only because it would be funny.

**Comment by Emisse** [comment]
the cargo bay external airlocks arent externals locked

**Comment by LordEclipse** [comment]
Remove it from Techs too, give Salvage even more power over cargo.

**Comment by Emisse** [comment]
ok

**Comment by UbaserB** [comment]
How does QM help salv now if they cant open external doors?

**Comment by Emisse** [comment]
none of the doors in cargobay are external

**Comment by asperger-sind** [comment]
yea idc about the cmo thread, this is dumb because cargo is in very close proximity to space and QM should interact with salv more than less, to counter an argument by removing an okay feature is dumb

**Comment by Emisse** [comment]
none of the doors in cargo bay are external, this changes literally nothing for qm

File: Resources/Prototypes/Roles/Jobs/Cargo/cargo_technician.yml [hunk 1]
```diff
@@ -9,7 +9,6 @@
   access:
   - Cargo
   - Maintenance
-  - External
   extendedAccess:
   - Salvage
```
Comment (SonicHDC) [inline Resources/Prototypes/Roles/Jobs/Cargo/cargo_technician.yml:13]: Maybe Extended External Access?

**Comment by Anzarot121** [comment]
How will they open the shuttle airlocks? They have external access.

**Comment by Emisse** [comment]
the doors automatically open when it docks

**Comment by Emisse** [comment]
i will change them to cargo lock though

**Pull request merged**